src/ansys/fluent/core/solver/flobject.py

```diff
@@ -641,18 +641,23 @@ class Map(SettingsBase[DictStateType]):
     """A Map object represents key-value settings."""
 
 
+def _get_new_keywords(obj, kwds):
+    newkwds = {}
```

Comment:
Personally I like doing it this way:  `newkwds = dict()`

Reply:
Hi Dan, 
I will follow these instructions now onwards.
Thank you.